lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -245,6 +245,23 @@ onnxWaitEvent(onnxEvent event) {
   return ONNXIFI_STATUS_SUCCESS;
 }
 
+/// Query ONNXIFI event state without blocking.
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI
+  onnxGetEventState(
+    onnxEvent event,
+    onnxEventState* state) {
+  auto *glowEvent = reinterpret_cast<glow::onnxifi::EventPtr>(event);
```

Comment:
`static_cast` may be a better option. `onnxEvent` is just `void*`

Reply:
@Maratyszcza Seems like the majority (all) onnxifi definitions for (event, graph, backend, etc) are void*. Is it safe to assume that it's not going to change? (There are other places in this implementation where reinterpret_cast could be replaced by static_cast, I'll change that in a separate commit).